src/Data.cpp

```diff
@@ -226,6 +226,9 @@ void Data::ReleaseUnneededEPG()
         // notify about the epg change...and delete it
         EPG_TAG tag;
         memset(&tag, 0, sizeof(EPG_TAG));
+        tag.iSeriesNumber = -1;
```

Comment:
Shouldn't we use directly your's PR `EPG_TAG_INVALID_SERIES_EPISODE` constant here?